database/availabilities.py

```diff
@@ -0,0 +1,57 @@
+from bson import ObjectId
+
+from database import collection_availabilities
+from models.models import Availability
+
+"""
+db integrity:
+- schedule delete - delete all schedule's availabilities
+- operator delete - delete all operator's availabilities
+- add availability - check for no duplications - NOT CHECKING
+"""
+
+
+# Schedule and operator ids are optional
+def get_availabilities(schedule_id: str, operator_id: str):
+    if schedule_id:
+        if operator_id:
+            # schedule and operator
+            result = collection_availabilities.find(
+                {"schedule_id": schedule_id, "operator_id": operator_id})
+        else:
+            # only schedule
+            result = collection_availabilities.find({"schedule_id": schedule_id})
+    elif operator_id:
+        # only operator
+        result = collection_availabilities.find({"operator_id": operator_id})
+    else:
+        # neither
+        result = collection_availabilities.find({})
```

Comment:
`query = {} | {"schedule_id: schedule_id} if schedule_id else {} | {"operator_id: operator_id} if operator_id else {}}
result = collection_availabilities.find(query)`

Reply:
did this:
    query = {}
    query.update({"schedule_id": schedule_id} if schedule_id else {})
    query.update({"operator_id": operator_id} if operator_id else {})
    result = collection_requests.find(query)